molecule/default/tests/test_dashboard.py

```diff
@@ -4,22 +4,51 @@
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
 @pytest.mark.jira('asc-222')
+# Experimental test
+def test_find_openrc_on_root(host):
+    assert host.file("/root/openrc").exists
+
+
+# Experimental test
+def test_find_openrc_on_opt(host):
+    assert host.file("/opt/openrc").exists
+
+
+# Experimental test
+def test_find_openrc_on_etc(host):
+    assert host.file("/etc/openrc").exists
+
+
+# Experimental test
 def test_cinder_service(host):
-    """Test to verify that cinder service is running on the cinder nodes
-
-    Args:
-        host(testinfra.host.Host): A hostname in dynamic_inventory.json/molecule.yml
-    """
-    if not host.file('/root/openrc').exists:
-        pytest.skip('openrc file not found')
-    elif host.file('/root/openrc').exists:
+
+    # fail test immediately if no cinder client on the host
+    assert host.exists("cinder")
+    if host.file('/root/openrc').exists:
```

Comment:
You are conflating two tests here and neither one is reflected in the test name anymore. Please make sure your concerns are atomic, reflected in separate tests, and named appropriately.
1. Test that the `cinder` command exits.
2. Test the `cinder service-list` contains "cinder-volume"

Reply:
yes, I did it on purpose to reduce the number of failed tests in the logs, if `cinder` command does not exist in a host, then there is no way to run `cinder service-list` on that host.

Reply:
ok, please disregard my previous comment.

Reply:
at the line 31, test only run when there is `/root/openrc` file. tests for existing of /root/openrc are done in in different tests